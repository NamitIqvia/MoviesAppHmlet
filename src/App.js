import './App.css';
import Movie from './components/movie/Movie';
import MovieDetails from './components/movie/MovieDetails';

import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';

function App() {
  return (
    <Router>
      <div className="App">
        <Switch>
            <Route path="/" exact render={() => <Movie />} />
            <Route path="/details" exact render={() => <MovieDetails />} />
        </Switch>
      </div>
    </Router>
  
  );
}

export default App;
