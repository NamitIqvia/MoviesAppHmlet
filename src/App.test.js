import React from 'react';
//import { Router } from 'react-router-dom';
import { BrowserRouter, Route } from 'react-router-dom';
import  App  from './App';
import renderer from 'react-test-renderer';




it('should take a snapshot', () => {
    
    const tree = renderer.create(<BrowserRouter><App /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
});