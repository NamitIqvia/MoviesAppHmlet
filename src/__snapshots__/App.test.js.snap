// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<div
  className="App"
>
  <h1
    className="app-heading"
  >
    Find your favourite movies below!!!
  </h1>
  <form
    onSubmit={[Function]}
  >
    <input
      className="search-box"
      data-testid="search bar"
      onChange={[Function]}
      placeholder="Enter the movie name here....."
      type="text"
      value=""
    />
  </form>
  <div />
  <div />
  <div
    className="movieDisplay"
  />
</div>
`;
